0_download_data/correctionutils.py

```diff
@@ -0,0 +1,114 @@
+"""
+This file contains functions to perform the splitting, cropping, and conversion of the second NF1 dataset using CellProfiler and
+to reorder and rename the metadata of the images to fit the standard from the pilot dataset.
+"""
+import os
+import pathlib
+from pathlib import Path
+
+
+def correct_images(path_to_pipeline: str, path_to_output: str, path_to_images: str):
+    """
+    correct NF1 second plate with CellProfiler
+
+    Parameters
+    ----------
+        path_to_pipeline (str): path to the CellProfiler .cppipe file
+        path_to_output (str): path to the output folder for the corrected images
+        path_to_images (str): path to folder with raw second plate images
+    """
+    if len(os.listdir(path_to_output)) == 0:
+        # run CellProfiler on a plate that has not been analyzed yet (e.g. no files)
+        command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}"
+        os.system(command)
+    else:
+        print("This plate has already been ran through CellProfiler!")
```

Comment:
Minor suggestion for clarity.

```suggestion
        print("This plate has already been processed by CellProfiler!")
```

Reply:
That sounds way better. Fixed!